### ISSUE TITLE
sign((-1)//typemin(Int)) != sign(typemin(Int)//(-1))

### ISSUE THREAD

**Issue by JeffreySarnoff** [JuliaLang/julia#32443]
```
julia> sign(typemin(Int)//(-1))
-1//1
julia> sign((-1)//typemin(Int))
1//1
```
and also for the other Signed bitstypes and also for `signbit`.

**Comment by Moelf** [comment]
problems seems to be:
```
julia> ((-1)//typemin(Int)).num
1

julia> (typemin(Int)//(-1)).num
-9223372036854775808
```

**Comment by musm** [comment]
This is because
 `typemin(Int) == -typemin(Int) # true`

**Comment by JeffreySarnoff** [comment]
`one(T)//typemin(T), typemin(T)//one(T)` are legit
`(-one(T))//typemin(T), typemin(T)//(-one(T))` are not because there is no value in T == |typemin(T)|
_we trap Rational overflow rather than autowiden, yet this is not trapped_

**Comment by Liozou** [comment]
Fixed by #32572?
